ur_robot_driver/launch/ur5_bringup.launch

```diff
@@ -5,7 +5,7 @@
   <arg name="reverse_port" default="50001" doc="Port that will be opened by the driver to allow direct communication between the driver and the robot controller."/>
   <arg name="script_sender_port" default="50002" doc="The driver will offer an interface to receive the program's URScript on this port. If the robot cannot connect to this port, `External Control` will stop immediately."/>
   <arg name="tf_prefix" default="" doc="tf_prefix used for the robot."/>
-  <arg name="controllers" default="joint_state_controller scaled_pos_joint_traj_controller speed_scaling_state_controller force_torque_sensor_controller" doc="Controllers that are activated by default."/>
+  <arg name="controllers" default="joint_state_broadcaster scaled_pos_joint_traj_controller speed_scaling_state_controller force_torque_sensor_controller" doc="Controllers that are activated by default."/>
```

Comment:
I vote to delete some of these args from urX_bringup.launch and let the defaults in ur_common.launch take care of them

Reply:
^ @destogl and @gavanderhoorn I would appreciate comments here.

Currently many of the args are passed like this:

urX_bringup.launch --> ur_common.launch --> ur_control.launch

I think we should define the defaults in ur_control.launch and delete the redundant definitions in the other 2 files. Less mindless passing of args is a good thing.

Reply:
I went ahead and pushed a commit to only define each arg default in one place. We can revert if you feel strongly

Reply:
`ur_common.launch` and `ur_control.launch` are not use at all... These are the leftovers from ROS1. Didn't want to delete those in this PR since it is huge...

Oh, this line change happened by find-replace all. And I wasn't careful when commiting.